client/landing/stepper/declarative-flow/connect-domain.ts

```diff
@@ -60,10 +61,13 @@ const connectDomain: Flow = {
 		const redirectTo = encodeURIComponent(
 			`/setup/${ flowName }/plans?domain=${ domain }&provider=${ provider }}`
 		);
-		const logInUrl =
-			locale && locale !== 'en'
-				? `/start/account/user/${ locale }?variationName=${ flowName }&pageTitle=Connect%20your%20Domain&redirect_to=${ redirectTo }`
-				: `/start/account/user?variationName=${ flowName }&pageTitle=Connect%20your%20Domain&redirect_to=${ redirectTo }`;
+
+		const logInUrl = useLoginUrl( {
+			variationName: flowName,
+			redirectTo: redirectTo,
```

Comment:
The function should encode the URL itself. And it does, because it uses `addQueryArgs`.